### ISSUE TITLE
take instructor into account

### ISSUE THREAD

**Issue by user2589** [cmu-student-government/cmunit#5]
In the feedback form we've got several suggestions to take instructors into account. I guess we can keep separate entries for all instructors teaching this course in the past 1 or 2 years, plus an average score across all instructors.

**Comment by michael-james** [comment]
Good idea. I would suggest we make any changes until we have a chance to reflect and regroup, mainly so that work does not have to be redone.
